packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1706,8 +1730,6 @@ class StackViewport extends Viewport implements IStackViewport {
         panCache as Point3
       );
 
-      // Restore rotation for the new slice of the image
-      this.rotationCache = 0;
```

Comment:
I have no idea why this was being done in the past, but I am pretty sure it is not correct.